### DISCUSSION TITLE
How to change the pivot point in keyframe mode?

### DISCUSSION THREAD

**Discussion by Firefly65536** [keiwando/toonsquid#142] [Q&A]
How can I change the pivot point mid-way through an animation? The keyframe diamond shape is blocking the pivot point so every time I move it, it moves the diamond instead. Split drawing doesn’t work since it messes up the keyframes (removes it)

**Comment by keiwando** [comment]
Since the pivot itself does not support keyframes, you could go to a frame on which there is no position keyframe already and then edit the pivot [as described here](https://toonsquid.com/handbook/transform/transform_properties/#editing-pivot). That will also insert a new position keyframe on that frame but you can just delete that.

That being said, the generally expected workflow with the pivot is that you initially place it where you need it to be and only afterwards add more position keyframes to the layer, since changes to the pivot will change the position of the contents of your layer on all frames. If you add position keyframes first and then change the pivot, you will most likely have to go back and adjust those keyframes as well.
